devel/pkgconfig/Portfile

```diff
@@ -67,7 +52,7 @@ configure.args      --disable-silent-rules \
 
 if {$subport eq "${name}-bootstrap"} {
     # search bootstrap prefix and then normal prefix
-    configure.args-append   --with-pc-path=${myprefix}/lib/pkgconfig:${myprefix}/share/pkgconfig:${prefix}/lib/pkgconfig:${prefix}/share/pkgconfig
+    configure.args-append   --with-pc-path=${prefix}/lib/pkgconfig:${prefix}/share/pkgconfig:${prefix}/lib/pkgconfig:${prefix}/share/pkgconfig
```

Comment:
This branch of the if statement will never be taken, since pkgconfig-bootstrap no longer exists.